webapp/WEB-INF/templates/admin/features/assign_users_right.html

```diff
@@ -51,35 +39,25 @@
 							maxlength=100 size=''
 							placeHolder='#i18n{portal.users.manage_users.search_users.labelEmail}' />
 					</@formGroup>
-					<@formGroup title='#i18n{portal.users.manage_users.search_users.labelStatus}' labelFor='status'
-						rows=2>
-						<@select name='status' id='status' size=''>
-							<#switch search_admin_user_filter.status>
-								<#case 0>
-									<option value="noValue">#i18n{portal.users.manage_users.search_users.labelAllStatus}
-									</option>
-									<option value="0" selected="selected">
-										#i18n{portal.users.create_user.statusLabelEnabled}</option>
-									<option value="1">#i18n{portal.users.create_user.statusLabelDisabled}</option>
-									<#break>
-										<#case 1>
-											<option value="noValue">
-												#i18n{portal.users.manage_users.search_users.labelAllStatus}</option>
-											<option value="0">#i18n{portal.users.create_user.statusLabelEnabled}
-											</option>
-											<option value="1" selected="selected">
-												#i18n{portal.users.create_user.statusLabelDisabled}</option>
-											<#break>
-												<#default>
-													<option value="noValue" selected="selected">
-														#i18n{portal.users.manage_users.search_users.labelAllStatus}
-													</option>
-													<option value="0">#i18n{portal.users.create_user.statusLabelEnabled}
-													</option>
-													<option value="1">
-														#i18n{portal.users.create_user.statusLabelDisabled}</option>
-													<#break>
-							</#switch>
+					<@formGroup title='#i18n{portal.users.manage_users.search_users.labelStatus}' rows=2 labelKey='' labelFor='status'>
+						<@select id='status' name='search_status' size=''>
+						<#switch search_admin_user_filter.status>
+							<#case 0>
+								<@option value='noValue' label='#i18n{portal.users.manage_users.search_users.labelAllStatus}' />
+								<@option value='0' selected=true label='#i18n{portal.users.create_user.statusLabelEnabled}' />

```

Comment:
According to w3c, use of the "selected" attribute should be :  
- selected
- selected="selected" (for XHTML compliance)